### PR TITLE
Fxing warnings and TODOs

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableGrpcClientOptions.java [hunk 1]
```diff
@@ -22,16 +22,25 @@
  */
 public class BigtableGrpcClientOptions {
 
+  private static int DEFAULT_STREAMING_BUFFER_SIZE = BigtableGrpcClient.SCANNER_BUFFER_SIZE;
+  // We can timeout when reading large cells with a low value here. With a 10MB
+  // cell limit, 60 seconds allows our connection to drop to ~170kbyte/s. A 10 second
+  // timeout requires 1Mbyte/s
+  private static int DEFAULT_READ_PARTIAL_ROW_TIMEOUT_MILLIS =
```
Comment (mgarolera) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableGrpcClientOptions.java]: 60 * 1000? I remember we were doing something like this in other PRs. Also MILLIS -> MS

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableRetriesExhaustedException.java [hunk 2]
```diff
@@ -21,6 +21,8 @@
  * An Exception that is thrown when an operation fails, even in the face of retrues.
  */
 public class BigtableRetriesExhaustedException extends IOException {
+  private static final long serialVersionUID = 6905598607595217072L;
```
Comment (mgarolera) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableRetriesExhaustedException.java]: Why?

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/IOExceptionWithStatus.java [hunk 3]
```diff
@@ -36,6 +37,7 @@ public IOExceptionWithStatus(String message, OperationRuntimeException cause) {
   /**
    * Status from the provided OperationRuntimeException.
    */
+  @SuppressWarnings("deprecation")
```
Comment (mgarolera) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/IOExceptionWithStatus.java]: Hmm.. why? As I read it, StatusRuntimeException shouldnt trigger deprecation warnings. Which one should be used instead?

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/FuzzyRowFilterAdapter.java [hunk 4]
```diff
@@ -72,6 +72,7 @@ private static RowFilter createSingleRowFilter(byte[] first, byte[] second) thro
       }
     }
     ByteString quotedValue = ByteString.copyFrom(baos.toByteArray());
+    quotingStream.close();
```
Comment (mgarolera) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/FuzzyRowFilterAdapter.java]: nit: Could this be done with a try with resource?

File: bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java [hunk 5]
```diff
@@ -220,10 +222,12 @@ protected BigtableClient getBigtableClient(
       TransportOptions dataTransportOptions,
       ChannelOptions channelOptions,
       ExecutorService executorService) {
-
     try {
-      return BigtableGrpcClient.createClient(
-          dataTransportOptions, channelOptions, executorService);
+      RetryOptions retryOptions = channelOptions.getUnaryCallRetryOptions();
+      BigtableGrpcClientOptions clientOptions =
```
Comment (mgarolera) [inline bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java]: This seems convoluted. Why is retryoptions in both channeloptions and clientoptions?

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableRetriesExhaustedException.java (hunk 2 above)
Comment (sduskis) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableRetriesExhaustedException.java]: All Java exceptions are Serializable.  The JVM spits out warnings if serialVersionUID isn't there.

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/IOExceptionWithStatus.java (hunk 3 above)
Comment (sduskis) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/IOExceptionWithStatus.java]: StatusRuntimeException is the correct class to use, and getStatus() is the right method.  I'll explain in person.  This is kind of what we have to do right now.

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/FuzzyRowFilterAdapter.java (hunk 4 above)
Comment (sduskis) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/FuzzyRowFilterAdapter.java]: Try with resource didn't look right in this case.  I tried it out.

File: bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java (hunk 5 above)
Comment (sduskis) [inline bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java]: It is convoluted.  I'll fix this in another PR.

**Comment by sduskis** [comment]
I address the comments, but didn't actually change code.  Can you PTAL?

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/IOExceptionWithStatus.java (hunk 3 above)
Comment (mgarolera) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/IOExceptionWithStatus.java]: I would put some comment here on why it is ok to ignore the deprecation warning. Warnings are there for a reason, so we should have a reason to suppress them, or talk to grpc if the warning is wrong.

Asides from this and the MS/MILLIS comment, LGTM.

**Pull request merged**